static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
```

Comment:
i guess my worry is that people add bespoke colours to suit their use case and the palette becomes irrelevant. these must be derived somehow from it?